app/src/utils/utils.js

```diff
@@ -217,5 +217,24 @@ module.exports = {
     });
 
     return objects;
+  },
+
+  /**
+   * Substitutes aliases in a instance definition by their value.
+   *
+   * @param {Object} instanceDef - The Instance Definition on which to apply the substitution
+   *
+   * @return {Object} substitutedInstanceDef - The Instance Definition with its aliases substituted by their actual value
+   */
+  substituteAliases: function(instanceDef) {
```

Comment:
Since we can't rule out that we might want to substitute aliases for other objects in the future, this should probably be renamed in references to instances or instance defs:
- `substituteInstanceAliases`
- `substituteInstanceDefinitionAliases`
- `resolveInstanceAliases`
- `resolveInstanceDefinitionAliases`

I guess i prefer any of the shorter ones.

Reply:
That was my inital naming acutally, but then thought it would be too long. I can change that back to one of your suggestions. 1st one I guess is the best compromise.